app/src/main/java/com/example/life_is_well/BMICalc.kt

```diff
@@ -1,11 +1,83 @@
 package com.example.life_is_well
 
+import android.content.Context
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.app.NotificationCompat.getColor
+import androidx.core.content.ContextCompat
 
 class BMICalc : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_bmicalc)
+
+        val weightText = findViewById<EditText>(R.id.etWeight)
+        val heightText = findViewById<EditText>(R.id.etHeight)
+        val calcButton = findViewById<Button>(R.id.btnCalculate)
+
+        calcButton.setOnClickListener {
+            val weight = weightText.text.toString()
+            val height = heightText.text.toString()
+            if(validateInput(weight, height)){
+                // BMI equation for Standard Measurement System (Pounds & Inches)
+                val bmi = (weight.toFloat()/((height.toFloat())*(height.toFloat()))) * 703
+                // Setting the result to two decimal places, Stackoverflow came in clutch here
```

Comment:
Don't be afraid to put the link of the Stackoverflow right in the comment. Could prove to be helpful to others and can't do any harm! 👍🏼

Reply:
@jacobburgo Will do moving forward.